packages/boot/tools/supports.ts

```diff
@@ -289,6 +289,14 @@ export const matchIter = (t: AvaT, iter, valueRef) => {
   matchValue(t, iter.value, valueRef);
 };
 
+export const AckBehavior = {
+  /** inbound responses are queued. use `flushInboundQueue()` to simulate the remote response */
+  Queued: 'QUEUED',
+  /** inbound messages are delivered immediately */
+  Immediate: 'IMMEDIATE',
+} as const;
+type AckBehaviorType = (typeof AckBehavior)[keyof typeof AckBehavior];
```

Comment:
We have a way to make the type name be the same as the value name in JSDoc. In `.ts` one would usually do `enum`. I'm not sure how to do it in .ts without enum.